plugins/statistics.koplugin/main.lua

```diff
@@ -823,12 +823,16 @@ end
 --          monthly - show monthly
 function ReaderStatistics:getDatesFromAll(sdays, ptype)
     local results = {}
+    local year_begin, year_end, month_begin, month_end
     local now_t = os.date("*t")
     local from_begin_day = now_t.hour *3600 + now_t.min*60 + now_t.sec
     local now_stamp = os.time()
     local one_day = 86400 -- one day in seconds
-    local period_begin = now_stamp - ((sdays-1) * one_day) - from_begin_day
     local sql_stmt_res_book
+    local period_begin
```

Comment:
Just stick a ` --luacheck: ignore` behind it. I'm not sure why it claims it's never accessed when that happens two lines down. >_>

Reply:
Because I forgot use it here https://github.com/robert00s/koreader/blob/21018d0e9b63fda576d70414c512134b632a4136/plugins/statistics.koplugin/main.lua#L889
:)

Reply:
Hehe, well, that's why we have it. :+1:

Reply:
exactly :)